### PR TITLE
Governance - verify runtime upgrade page

### PR DESCRIPTION
**Pull request by lreesby**

### CODE REVIEW

**Comment by lreesby** [comment]
image
image
image
image

File: _data/navigation_docs.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@
   - on-chain-automation
   - identity
 
+- title: Governance
```
Comment (irsal) [inline _data/navigation_docs.yml:13]: I would comment this out for now since once you merge it will deploy to prod

**Review by irsal — APPROVED** [review]
one change, then lgtm

**Comment by lreesby** [comment]
`Upgrade Preimage Information` is shown twice in the images above but that's just the screen-shotting I did.

**Review by rhuttman — APPROVED** [review]

**Review by justinzhou93 — APPROVED** [review]

**Pull request merged**